openedx/features/assessment/helpers.py

```diff
@@ -71,7 +72,7 @@ def autoscore_ora(course_id, usage_key, student):
         student_item__student_id=anonymous_user_id,
         student_item__item_id=usage_key,
         student_item__item_type=ORA_BLOCK_TYPE
-    ).first()
+    ).order_by('id').first()
```

Comment:
please add a comment that in case of multiple submissions we need to pick the lates one.